### ISSUE TITLE
Donate page: "Other amount" field is awkward to use with Voice Assistant on Android

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#259]
## Description of issue

When using the Voice Assistant on Android, it gets a bit awkward trying to interact with the "Other amount" field. 

Firstly it's a bit confusing when you tap on it: “Edit box, double tap to edit, double tap to enter text, double tap and hold to long press”

Secondly, to actually interact with it, you need to double tap and hold. This is quite cumbersome. iOS only requires a double tap. 

## Expected behaviour

When you single tap on the "Other amount" field, it reads out "Other amount edit box, double tap to edit".
Then when you double tap, you can edit the field. 

## Steps to reproduce

- Enable the Voice Assistant
- Open the Donation page
- Tap on the Other amount field
> Reads out "Edit box, double tap to edit, double tap to enter text, double tap and hold to long press"
- Double-tap on Other amount field
> Doesn't work to edit field

## OS / Browser / Device model

Galaxy S8

**Comment by frigus02** [comment]
Yeah, I mentioned the same here: https://github.com/redbadger/pride-london-app/pull/186#issuecomment-385283290

I think this is a bug in React Native. It looked hard to fix to me 😞

**Comment by RGBboy** [comment]
High priority, but lets be pragmatic. If this takes too much effort we can park until later.

**Comment by frigus02** [comment]
I believe I found a fix. Will make a PR in the next days.
